src/cgame/default/cg_entity_trail.c

```diff
@@ -456,15 +456,15 @@ static void Cg_LightningTrail(cl_entity_t *ent, const vec3_t start, const vec3_t
 			return;
 
 		cgi.ColorFromPalette(12 + (Random() & 3), p->part.color);
-		Vector4Set(p->color_vel, 0.0, 0.0, 0.0, -100.0);
+		Vector4Set(p->color_vel, 0.0, 0.0, 0.0, -100);
 
 		p->part.scale = 8.0;
 		p->part.scroll_s = -8.0;
 
 		VectorCopy(pos, p->part.org);
 
 		if (dist <= 48.0) {
-			VectorScale(dir, dist, delta);
+			VectorScale(dir, -dist, delta);
 			offset = 0.0;
```

Comment:
Yep, caused lightning bolt at end of chain to go backwards. Now you can shoot lightning bolt at a close wall and still see zappyzap